### PR TITLE
Documentation for Taipy Studio Configuration Builder (#319)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #319.

### CODE REVIEW

File: docs/manuals/studio/config.md [hunk 1]
```diff
@@ -1,8 +1,188 @@
...
+This panel is split into several sections that let the user access all the
+configuration elements:
+
+- Config Files: holds a list of configuration files (`*.toml`) that exist in the
+    project.<br/>
+    The user can select a configuration file in this list to populate the other
+    sections of the Taipy Configs view.
+- Data Notes: holds the list of
+    [Data Node configurations](../core/config/data-node-config.md) defined
+    in the selected configuration files.<br/>
```
Comment (FredLL-Avaiga) [inline docs/manuals/studio/config.md]: files => file

File: docs/manuals/studio/config.md [hunk 2]
```diff
@@ -1,8 +1,188 @@
...
+    The user can select a Data Node configuration in this list, to access all its
+    properties in the Details section (see below).
+- Tasks: holds the list of
+    [Task configurations](../core/config/task-config.md) defined
+    in the selected configuration files.<br/>
```
Comment (FredLL-Avaiga) [inline docs/manuals/studio/config.md]: idem

File: docs/manuals/studio/config.md [hunk 3]
```diff
@@ -1,8 +1,188 @@
...
+elements, as well as change their properties in the "Details" section.
+
+All your actions impact the selected configuration file;
+you can use Visual Studio Code's undo/redo feature if needed.
+
+## Managing configuration files
+
+The "Config Files" section holds the list of configuration files (`*.toml`) in your
+project. You will select the one you want to edit from that list.<br/>
+This list shows the root names of all the configuration files. If two configuration files
```
Comment (FredLL-Avaiga) [inline docs/manuals/studio/config.md]: root name => base name ?

File: docs/manuals/core/concepts/data-node.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ A _data node_ can reference any data:
 - a text,
 - a numeric value,
 - a list, tuple, set, dictionary of parameters,
-- a custom python object (a dataframe, a dataclass, etc.),
+- a custom Python object (a Pandas DataFrame, a dataclass, etc.),
```
Comment (jrobinAV) [inline docs/manuals/core/concepts/data-node.md]:
```suggestion
- a custom Python object (a data frame, a data class, etc.),
```

File: docs/manuals/studio/config.md [hunk 5]
```diff
@@ -1,8 +1,188 @@
...
+Similarly, if you remove or rename a configuration file from the Explorer panel of Visual
+Studio Code, the change is immediately reflected in the Config Files section of the
+Taipy Configs panel.
+
+Note that if you right-click a configuration file, Taipy Studio displays a menu that
+let you choose one of two options:
+
+- "Reveal file in explorer view": Visual Studio Code will select and focus on the file
+    item in the project's files in the Explorer pane.
+- "Show view": creates a view on the configuration file that represents the entire content
```
Comment (FredLL-Avaiga) [inline docs/manuals/studio/config.md]: on the configuration file => from the configuration file ?
Add the location of the created view ? editor center space ...

File: docs/manuals/studio/config.md [hunk 6]
```diff
@@ -1,8 +1,188 @@
...
+  <img src="../images/config_autocomplete.gif" width=80%>
+</p>
+
+You will also get support for auto-completion in other contexts, such as a reference
+to another configuration element.
+
+### Spotting problems
+
+Taipy Studio helps point out potential problems in your configuration files that are
+indicated both in the body of the file content and in the Problems pane.
```
Comment (FredLL-Avaiga) [inline docs/manuals/studio/config.md]: and in the config details view

**Review by FredLL-Avaiga — APPROVED** [review]
Looks great so far

File: docs/manuals/studio/config.md [hunk 7]
```diff
@@ -1,8 +1,188 @@
...
+Applications that rely on [Taipy Core](../core/index.md) need to define
+a [Configuration](../core/config/index.md) where the data flow is
+described.<br/>
+This configuration can be coded in Python, tuned using environment variables,
+or be expressed as a TOML file as described in the
+[TOML file configuration](../core/config/advanced-config.md#explicit-toml-file-configuration)
+section.
 
-## ...
+Taipy Studio provides ways to design the Taipy Configuration files for your applications.
```
Comment (jrobinAV) [inline docs/manuals/studio/config.md]:
```suggestion
Taipy Studio provides graphical ways to design the Taipy Configuration files for your applications.
```

File: docs/manuals/studio/config.md [hunk 8]
```diff
@@ -1,8 +1,188 @@
 # Editing configuration files
 
-[TODO]
+Applications that rely on [Taipy Core](../core/index.md) need to define
+a [Configuration](../core/config/index.md) where the data flow is
+described.<br/>
+This configuration can be coded in Python, tuned using environment variables,
+or be expressed as a TOML file as described in the
+[TOML file configuration](../core/config/advanced-config.md#explicit-toml-file-configuration)
+section.
```
Comment (jrobinAV) [inline docs/manuals/studio/config.md]: Two remarks: 

1. Vincent and I recently made some updates to the page you are referring to. I just pushed the changes. You link is no longer up-to-date.
 
2. What you are saying is not really the truth. The section you mention is made to document how to override a python code configuration using TOML file, And not how to build a complete configuration in TOML.

I would not say that we can build a configuration using TOML. I would just say either we use Python code, or we use Studio and load the TOML result generated by Studio.

```suggestion
This configuration can be coded in Python, tuned using environment variables
(as described in the [advanced configuration](../core/config/advanced-config.md#Python-code-configuration) section).
```

**Review by jrobinAV — COMMENTED** [review]
1. OK with 'python' -> 'Python' and 'dataframe' to 'DataFrame' | 'data frame'.
2. a remark concerning config builder documentation and the reference to Core user man.

File: docs/manuals/studio/config.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/config.md]: 👌

File: docs/manuals/studio/config.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/config.md]: idem

File: docs/manuals/studio/config.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/config.md]: Yes! That's far better thanks.

File: docs/manuals/studio/config.md (hunk 6 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/config.md]: Ok!

**Review by drake-stamos — APPROVED** [review]
LGTM

**Pull request merged**